include/stm32f4xx_hal_conf.h

```diff
@@ -152,7 +152,7 @@
 #define  INSTRUCTION_CACHE_ENABLE     1U
 #define  DATA_CACHE_ENABLE            1U
 
-#define  USE_HAL_ADC_REGISTER_CALLBACKS         0U /* ADC register callback disabled       */
+#define  USE_HAL_ADC_REGISTER_CALLBACKS         1U /* ADC register callback disabled       */

```

Comment:
If the ADC is out, should this not be removed?

Reply:
Yep, didn't see all the extra stuff hiding in this file.